ansible/roles_ocp_workloads/ocp4_workload_project_request_template/templates/project_request_template.j2

```diff
@@ -12,7 +12,7 @@ objects:
     namespace: ${PROJECT_NAME}
   spec:
     limits:
-      {{ ocp4_workload_project_request_template_limits | to_yaml }}
+      {{ ocp4_workload_project_request_template_limits }}
```

Comment:
I would write

```yaml
  limits: {{ ocp4_workload_project_request_template_limits | to_json }}
```

i don't know about the other notation

according to my test the output is a bit different. 

```
    - debug:
        msg: |
          instances:
            {{ instances }}
    - debug:
        msg: |
          instances: {{ instances|to_json}}
```

produces:

```
TASK [debug] **************************************************************************************************************************************
Thursday 07 May 2020  22:13:19 +0200 (0:00:00.160)       0:00:00.340 ********** 
ok: [localhost] => 
  msg: |-
    instances:
      [{'name': 'bastion', 'count': 1, 'unique': True, 'alt_name': 'bastion', 'floating_ip': True, 'metadata': [{'AnsibleGroup': 'bastions,clientvms'}, {'function': 'bastion'}, {'user': 'nate'}, {'ostype': 'linux'}], 'rootfs_size': 30, 'network': 'ocp', 'security_groups': ['bastion_sg']}, {'name': 'utilityvm', 'count': 1, 'floating_ip': False, 'flavor': None, 'metadata': [{'AnsibleGroup': 'utility'}, {'function': 'bastion'}, {'user': 'nate'}, {'ostype': 'linux'}], 'rootfs_size': 50, 'network': 'ocp', 'security_groups': ['utility_sg']}, {'name': 'utilityvm2', 'count': 1, 'floating_ip': False, 'flavor': None, 'metadata': [{'AnsibleGroup': 'utility'}, {'function': 'bastion'}, {'user': 'nate'}, {'ostype': 'linux'}], 'rootfs_size': 50, 'network': 'ocp'}]

TASK [debug] **************************************************************************************************************************************
Thursday 07 May 2020  22:13:19 +0200 (0:00:00.159)       0:00:00.500 ********** 
ok: [localhost] => 
  msg: |-
    instances: [{"name": "bastion", "count": 1, "unique": true, "alt_name": "bastion", "floating_ip": true, "metadata": [{"AnsibleGroup": "bastions,clientvms"}, {"function": "bastion"}, {"user": "nate"}, {"ostype": "linux"}], "rootfs_size": 30, "network": "ocp", "security_groups": ["bastion_sg"]}, {"name": "utilityvm", "count": 1, "floating_ip": false, "flavor": null, "metadata": [{"AnsibleGroup": "utility"}, {"function": "bastion"}, {"user": "nate"}, {"ostype": "linux"}], "rootfs_size": 50, "network": "ocp", "security_groups": ["utility_sg"]}, {"name": "utilityvm2", "count": 1, "floating_ip": false, "flavor": null, "metadata": [{"AnsibleGroup": "utility"}, {"function": "bastion"}, {"user": "nate"}, {"ostype": "linux"}], "rootfs_size": 50, "network": "ocp"}]
```

Reply:
I tried it. It works.